js/lbank.js

```diff
@@ -625,6 +716,9 @@ module.exports = class lbank extends Exchange {
             this.checkRequiredCredentials ();
             const query = this.keysort (this.extend ({
                 'api_key': this.apiKey,
+                'timestamp': this.milliseconds (),
+                'signature_method': 'RSA',
+                'echostr': 'P3LHfw6tUIYWc8R2VQNy0ilKmdg5pjhbxC7',
```

Comment:
A quick question: where does this `echostr` come from?

Reply:
On their API doc page (https://docs.lbkex.net/en/#authentication) you can read a description regarding 'echostr'

_echostr: the param is digit or letter，length is from 30 to 40. You can directly use echostr of SDK, it's safe._

That's what I did. I used the `echostr` from SDK.